pulsar-common/src/test/java/org/apache/pulsar/common/policies/data/OffloadPoliciesTest.java

```diff
@@ -432,4 +433,16 @@ private byte[] loadClassData(String name) throws IOException {
         }
     }
 
+    @Test
+    public void testCreateOffloadPoliciesWithExtraConfiguration() {
+        Properties properties = new Properties();
+        properties.put("managedLedgerOffloadExtraConfigKey1", "value1");
+        properties.put("managedLedgerOffloadExtraConfigKey2", "value2");
+        OffloadPoliciesImpl policies = OffloadPoliciesImpl.create(properties);
```

Comment:
```
Properties properties = new Properties();
Properties extraProps = new Properties();
extraProps.put("configKey1", "value1");
extraProps.put("configKey2", "value2");
properties.put("managedLedgerOffloadExtraConfig", extraProps);
OffloadPoliciesImpl policies = OffloadPoliciesImpl.create(properties);
```
Seems better then `prefix` keys

Reply:
The properties come from the ServiceConfiguration, so we need a prefix to avoid bringing too many unused properties into the offloader.